First draft of slack info

djphan opened the pull request:
To address https://github.com/devedmonton/devedmonton.com/issues/99

First pass at it. Simple design but added colour. Don't have graphic skills if we wanted something custom there but thoughts?

![Screen Shot 2021-10-26 at 8 01 28 PM](https://user-images.githubusercontent.com/6673572/138987295-4f91dd53-a13a-47da-a39c-b8149ef3c247.png)

djphan commented on pages/index.vue:
```diff
@@ -187,8 +187,123 @@
+                            </h3>
+                            <ul class="text-l p-6 text-gray-500">
+                                <li>#introductions to say "Hi"</li>
+                                <li>
```
Updated
![Screen Shot 2021-10-31 at 10 24 55 PM](https://user-images.githubusercontent.com/6673572/139621505-364d80ce-a5c2-4ef0-a903-afd53b4682de.png)
.

MandyMeindersma replied:
How hard is it to remove the box around the slack logo?

That's my only concern! Other than that it looks amazing

djphan replied:
It's not hard to remove the box. I just need to remove the styles and will do so now.

djphan commented on pages/index.vue:
```diff
@@ -187,8 +187,123 @@
+                                Channels to Checkout
+                            </h3>
+                            <ul class="text-l p-6 text-gray-500">
+                                <li>#introductions to say "Hi"</li>
```
Updated
![Screen Shot 2021-10-31 at 10 24 55 PM](https://user-images.githubusercontent.com/6673572/139621538-13ac1e20-2120-48eb-89ac-4624fc14b91e.png)
.

djphan commented on pages/index.vue:
```diff
@@ -187,8 +187,123 @@
+                        </p>
+
+                        <p class="text-gray-500">
+                            By joining this Slack you agree to abide by our
```
Updated

djphan commented on pages/index.vue:
```diff
@@ -663,4 +778,16 @@ export default {
     padding-top: 15px;
 }
+
+#slack-message-block {
```
> Instead of the "useful links"
> 
> Can we have the slack thing like at the top
> 
> ![image](https://user-images.githubusercontent.com/17459171/139131316-2fd71fde-3782-4491-bfee-3392ef5d9f4a.png)

Regarding this I changed the link to go to the anchor tag like so..
![Screen Shot 2021-10-31 at 10 25 00 PM](https://user-images.githubusercontent.com/6673572/139621593-013406bd-29c8-4a49-a045-357ab3cf22a5.png)
.

MarkBennett commented:
A new user left a comment. This user must be approved by a Netlify team owner before comments can be displayed.

[Approve this user](https://app.netlify.com/teams/devedmonton/reviewers)

MandyMeindersma commented:
@MarkBennett  it doesn't let me reply to your comment. 

I am confused about where you want these things?

Cause the code of conduct link is there in the description?

And random coffee's is down below in it's own section (unless you meant you would like it added to the channels list then yes I agree)

MarkBennett commented:
Does anyone else feel like Slack should really just have it's own page, rather than a section?

MandyMeindersma commented:
> Does anyone else feel like Slack should really just have it's own page, rather than a section?

Personally no, I think the biggest thing about us is the slack and I think it should be front and center?
I am scared less people would see it if they had to find the slack page. 

Plus when I tell people about DES I will give them the link to our home page so they can get to the slack. If I just send them the link to the slack page they will miss out on the homepage and all the other descriptions

MarkBennett commented:
> Personally no, I think the biggest thing about us is the slack and I think it should be front and center? I am scared less people would see it if they had to find the slack page.
> 
> Plus when I tell people about DES I will give them the link to our home page so they can get to the slack. If I just send them the link to the slack page they will miss out on the homepage and all the other descriptions

I'm thinking that we'd keep the Slack button front and centre at the top like it is now. Just change it so it links to it's own page rather than the section on the homepage.

MarkBennett commented:
> Plus when I tell people about DES I will give them the link to our home page so they can get to the slack. If I just send them the link to the slack page they will miss out on the homepage and all the other descriptions

I definitely agree on this. Telling people to just go to devedmonton.com to join is nice, and the Slack should be right at the top there since that's what most people are getting sent there for.

MarkBennett commented:
> Cause the code of conduct link is there in the description?

Yeah, I don't know how I missed that. Scratched that from the feedback. :P

MarkBennett commented:
> And random coffee's is down below in it's own section (unless you meant you would like it added to the channels list then yes I agree)

Yes, I think #random-coffee should be listed in the channels and at the bottom. It's a good funnel to help new people get started and meet the community, especially with so much being virtual and online.

djphan commented:
Apologies, took a week off from contributions to recharge. Going to go over comments now

djphan commented:
For @MarkBennett @MandyMeindersma 
I restored the button to join the Slack at the top. But there is a link to find out more info about the Slack and the menu option goes to the new /slack page as described above

![Screen Shot 2021-11-07 at 3 01 02 PM](https://user-images.githubusercontent.com/6673572/140663560-4835bd6b-0614-4e47-b613-451a7855eb8a.png)

For @MandyMeindersma I removed the shadow box for the slack joining section per feedback in this screenshot

For @MarkBennett I added the #random_coffees here but not sure what you're looking for for the meetup channels? I have a generic line about meetup specific channels for tech and languages but I can list a bunch. Originally since this was a section I referred to the meetup section but since its now in it's own page I refer to that master list. Thoughts?

![Screen Shot 2021-11-07 at 3 08 15 PM](https://user-images.githubusercontent.com/6673572/140663561-e5c2618e-7123-47c1-8d11-063322b1638c.png)

MandyMeindersma commented:
Hey @djphan  sorry to be so wishy washy on your pr. I really dislike the slack having it's own page but @MarkBennett  likes it. He did add this to the ticket though:
>  It's easy to move the Slack section to it's own page if the homepage gets to busy and it would be nice to get this out there! :)


I personally think it will be harder for new people to figure out about DES if the slack is on a different page (people will only link to the slack and it might be confusing for a new user)

So someone else can approve this if they want but I prefer the earlier iterations

MandyMeindersma commented:
If we do have to keep this though I think it might be better to have the homepage slack button (on the index page) to link to your slack info page, not the actual slack

djphan commented:
I moved the slack page back to the main page. With some feedback about resources getting their own pages I think this keeps the main page stream lined. Those PRs will be done later (probably after this one)

I think though if we have a difference of opinion on where this section should be we should try out one layout and gather feedback from users (especially the new members of the Slack). We can always change the code so it doesn't have to be right the first time but if people think it's hard to find how to join the Slack workspace then we can adjust the design.

@MandyMeindersma @MarkBennett thoughts? Below is the screenshot of what this looks like.

![Screen Shot 2021-11-21 at 2 51 31 PM](https://user-images.githubusercontent.com/6673572/142780258-3125997f-27e3-4af7-87be-31428929ada9.png)
![Screen Shot 2021-11-21 at 2 51 38 PM](https://user-images.githubusercontent.com/6673572/142780260-474d232b-136e-4486-9294-57bddd5bb7dd.png)

MandyMeindersma commented:
Looks great! Yeah if we want it to be another page then we can make another ticket. Does that sound okay @MarkBennett ?

MarkBennett commented:
Looks great to me to @MandyMeindersma. Nice work @djphan! 🙂

MandyMeindersma commented:
@all-contributors please add @djphan for code and design

Per spec at:

https://allcontributors.org/docs/en/emoji-key
